### DISCUSSION TITLE
CAN SOMEBODY PLEASE HELP ME, I NEED HELP WITH MY HOMEWORK!!! I DONT FIND THE ANSWER AND I'M LOOKING FOR WEEKS NOW

### DISCUSSION THREAD

**Discussion by hftwich** [community/community#39515] [New to GitHub]
i have to make a project, okay i know how to do that but than i have to make it public and i dont find the source to set it public, and when i have this i have to create a link so that my teacher can look @ my page with github, i have to share it to him. But the last time i have share a link, it doenst exist the page is showing this. 

so how can i find the source, and set this public and how can i create a link that hi can use to open my page with github. 

please otherwise i have to end my school, because i cant make any homework if dont know this! 

help me please!!!

**Comment by airtower-luna** [comment]
If you're looking how to make your repository public, see: [Setting repository visibility](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility)

If you mean something else, please clarify what you mean. And avoid using all caps (like in the title), it makes it seem like you're screaming at people, which is rude.

**Comment by ihnatenkoo** [comment]
Hello, all github repositories are public by default
If you need link to your repository you can find it in search bar (it consists of `github.com/YOUR_LOGIN/YOUR_REPOSITORY_NAME`
For example, link to your project15 is [https://github.com/hftwich/project15](https://github.com/hftwich/project15)
![gg](https://user-images.githubusercontent.com/89586345/202867005-e873705d-33e9-4227-8c01-6a68cb549f40.png)

**Comment by jfullstackdev** [comment]
you can simply add your teacher as collaborator and the repo will remain private

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

this way, you don't need to send the link, your teacher will be simply notified, you just mention it that you added him/her

**Reply by hftwich** [reply]
![github probleem](https://user-images.githubusercontent.com/115465180/206851105-6e4b3c2c-f1a8-43ab-8734-a830b9eb2ea1.jpg)
Goededag, bedankt voor je antwoord. 

Hij wil mijn site zien zoals ik hem heb gemaakt van buiten volgens mij, zodra ik dan de link naar hem stuur en die zelf open staat er dat de pagina niet bestaat of gevonden kan worden. Hele slechte begeleiding vanuit die thuisstudie instantie.

**Reply by jfullstackdev** [reply]
seems this is a good translation 

![image](https://user-images.githubusercontent.com/47092464/206852212-1aa31070-30e2-47a9-aa00-d8de9d80003b.png)

there should be no problem opening and accepting collaborator invitation, given the above steps followed

you can have the screenshot from your teacher so we can take a look what's happening there

**Reply by hftwich** [reply]
Hello,

than i get this.
![github probleem2](https://user-images.githubusercontent.com/115465180/206853010-106c2d79-4c70-4051-8988-91a91411fc85.jpg)

**Reply by jfullstackdev** [reply]
that is GitHub Pages and `owner.github.io` is not you or your teacher, right ? 

![image](https://user-images.githubusercontent.com/47092464/206853705-ddda9ae9-7ef8-45fa-9343-dd7869fe4de4.png)

there is this Org, so it's already taken

https://github.com/owner

this is the one you want to share 

![image](https://user-images.githubusercontent.com/47092464/206853775-876c226d-b5ca-43df-be2a-911d8cea1cbb.png)

https://hftwich.github.io/project15/

send it to your teacher
